src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -6,298 +6,318 @@ import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
 import createWrapper from '../../../lib/components/test-utils/selectors';
 import { viewports } from './constants';
 
+const testIf = (condition: boolean) => (condition ? test : test.skip);
+
 const wrapper = createWrapper().findAppLayout();
 
+interface SetupTestObj {
+  theme: string;
+  pageName: string;
+  splitPanelPosition?: string;
+  mobile: boolean;
+}
+
 function setupTest(
   testFn: (page: BasePageObject) => Promise<void>,
-  { pageName = 'with-split-panel', visualRefresh = false, splitPanelPosition = '', mobile = false }
+  { theme, pageName = 'with-split-panel', splitPanelPosition = '', mobile = false }: SetupTestObj
 ) {
   return useBrowser(async browser => {
-    const url = `#/light/app-layout/${pageName}?visualRefresh=${visualRefresh}${
-      splitPanelPosition ? `&splitPanelPosition=${splitPanelPosition}` : ''
-    }`;
     const page = new BasePageObject(browser);
+    const params = new URLSearchParams({
+      visualRefresh: `${theme.startsWith('visual-refresh')}`,
+      appLayoutWidget: `${theme === 'visual-refresh-toolbar'}`,
+      ...(splitPanelPosition
+        ? {
+            splitPanelPosition,
+          }
+        : {}),
+    });
     await page.setWindowSize(mobile ? viewports.mobile : viewports.desktop);
-    await browser.url(url);
+    await browser.url(`#/light/app-layout/${pageName}?${params.toString()}`);
     await page.waitForVisible(wrapper.findContentRegion().toSelector());
     await testFn(page);
   });
 }
 
-[true, false].forEach(visualRefresh =>
-  describe(`visualRefresh=${visualRefresh}`, () => {
-    [true, false].forEach(mobile =>
-      describe(`mobile=${mobile}`, () => {
-        test(
-          'should not focus panels on page load',
-          setupTest(
-            async page => {
-              await expect(page.isFocused('body')).resolves.toBe(true);
-            },
-            { pageName: 'with-split-panel', visualRefresh, mobile }
-          )
-        );
+describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)('%s', theme => {
+  [true, false].forEach(mobile =>
+    describe(`mobile=${mobile}`, () => {
+      test(
+        'should not focus panels on page load',
+        setupTest(
+          async page => {
+            await expect(page.isFocused('body')).resolves.toBe(true);
+          },
+          { pageName: 'with-split-panel', theme, mobile }
```

Comment:
There is still a human factor of not forgetting to pass this argument to every test. 

If would be safer to move the `setupTest` function from the top level under the `describe.each` loop and inherit it from the closure automatically

(I see you mitigated it with `SetupTestObj`, so not a blocker, but I still prefer when it is less code to write)

Reply:
I will address this in a follow up. Looking at the //TODOs there will be a few followup